components/map/MapPin.tsx

```diff
@@ -0,0 +1,41 @@
+import { View } from 'react-native'
+
+import MapPinSvg from '@/assets/ui-icons/map-pin.svg'
+import MapPinNoZoneSvg from '@/assets/ui-icons/map-pin-no-zone.svg'
+import { useScreenCenter } from '@/modules/map/hooks/useScreenCenter'
+
+import Typography from '../shared/Typography'
+
+interface Props {
+  price?: number
+}
+
+const PIN_WIDTH = 56
+const PIN_HEIGHT = 98
+
+const MapPin = ({ price }: Props) => {
+  const screenCenter = useScreenCenter()
+  const position = {
+    top: screenCenter.top - PIN_HEIGHT + 4,
+    left: screenCenter.left - PIN_WIDTH / 2,
+  }
+  const pinSize = { width: PIN_WIDTH, height: PIN_HEIGHT }
+
+  return (
+    <>
+      <View className="absolute items-center" style={position}>
+        {price ? <MapPinSvg {...pinSize} /> : <MapPinNoZoneSvg {...pinSize} />}
+        {price && (
+          <Typography variant="small-bold" className="absolute text-white">
+            {price} €
+          </Typography>
+        )}
+      </View>
+      <View className="absolute" style={{ top: screenCenter.top, left: screenCenter.left }}>
+        <View className="h-[1px] w-[1px] bg-[#f00]" />
```

Comment:
Just to let you know, that "tailwind values" for 1px size are `-px`, so `h-px w-px` is the syntax that works :)